nmap_dns.cc

```diff
@@ -784,7 +793,10 @@ static void read_evt_handler(nsock_pool nsp, nsock_event evt, void *) {
               }
               output_summary();
               stat_ok++;
+              /*Since we have got the answer we can avoid iterating over all answers*/
```

Comment:
@tremblerz It seems correct, but for clarity I would recommend not using "return" here but instead make `processing_successful` a condition of the `for` loop. That way we don't avoid any further processing that the function may want to do and the `switch` case can simply `break;` in the same way all the others do.